tests/ProxyConnectorTest.php

```diff
@@ -11,32 +11,29 @@ class ProxyConnectorTest extends AbstractTestCase
 {
     private $connector;
 
-    public function setUp()
+    /**
+     * @before
+     */
+    public function setUpMock()
     {
         $this->connector = $this->getMockBuilder('React\Socket\ConnectorInterface')->getMock();
     }
 
-    /**
-     * @expectedException InvalidArgumentException
-     */
     public function testInvalidProxy()
     {
+        $this->setExpectedException("InvalidArgumentException");
```

Comment:
Nit: Can you update this to use single quotes instead of double quotes? (see also below)